app/views/users/_sidebar.html.erb

```diff
@@ -5,10 +5,10 @@
   <li><%= span_if_current ts("Profile"), user_profile_path(@user) %></li>
   <% if @user.pseuds.size > 1 %>
     <li class="pseud" aria-haspopup="true">
-      <% pseud_link_text = current_page?(@user) ? ts("Pseuds") : (@author ? @author.name : @user.login) %>
+      <% pseud_link_text = current_page?(@user) ? ts("Pseuds") : (@pseud ? @pseud.name : @user.login) %>
```

Comment:
Hmm. I like the idea of having the current pseud name as the head of the pseud switcher (particularly since that seems to have been the original intention), but it does mean that on the page for a pseud, the username isn't displayed anywhere prominent.

For example, [testy's orphantest1 pseud](https://archiveofourown.org/users/testy/pseuds/orphantest1) has no works and no bookmarks, so the only place testy's name appears at the moment is at the top of the pseud switcher. You can see also see the testy pseud in the switcher, but only if you click on it. For pseuds with works and/or bookmarks, like [testy's orphantest2 pseud](https://archiveofourown.org/users/testy/pseuds/orphantest2), you can also see the username in the byline, but that's a lot easier to miss.

Of course, the current location of the username isn't as prominent as the pseud name, and I suspect that's the cause of a number of accidental subscriptions to orphan_account. But I think removing it from its current location (without finding somewhere else to put it) might make those types of mistakes more frequent.

Reply:
Hmm, interesting. I actually thought the current behavior was a bug -- I expected it to either show "Pseuds" all the time or the currently-selected pseud and couldn't fathom why it was showing something else.

I think if we're worried about confusion regarding which user's pseud we're looking at, it makes more sense to change the page headings to display "pseud (username)" -- since that's the format we prefer in most other places on the Archive -- rather than relying on someone figuring it out from the pseud switcher.

Reply:
Made an issue for fixing the dashboard's heading: https://otwarchive.atlassian.net/browse/AO3-6006

Reply:
Same: please let me know if this thread still needs resolving, and how.

Reply:
It can stay how you have it. We have a new issue for displaying the username more prominently.